c8-ascii-art.txt

```diff
@@ -0,0 +1,9 @@
+/* ________/\\\\\\\\\_        _____/\\\\\\\\\____         */
+/*  _____/\\\////////__        ___/\\\///////\\\__        */
+/*   ___/\\\/___________        __\/\\\_____\/\\\__       */
+/*    __/\\\_____________        __\///\\\\\\\\\/___      */
```

Comment:
I'd be tempted to just make this a template string rather than bothering to add a file.